def ExceptionError(value):
    if value is None:
        raise ValueError("Input não pode ser None")
    try:
        return int(value)
    except ValueError:
        raise ValueError("Entrada inválida. Digite um número inteiro válido.")

def compare():
    try:
        num_1 = ExceptionError(input('Digite um valor: '))
        num_2 = ExceptionError(input('Agora digite outro valor: '))
        
        if num_1 > num_2:
            print('O primeiro valor é maior que o segundo.')
        elif num_2 > num_1:
            print('O segundo valor é maior que o primeiro.')
        else:
            print('Ambos os valores são iguais.')
    
    except ValueError as e:
        print(f'Erro: {e}')

compare()
